### PR TITLE
RBAC and Settings Page Almost Complete, supabase auth CallerInvite Bug needs to be ironed out

### PR DESCRIPTION
**Pull request by manstephen**
Added SignUp button to the navbar, fixed a bad import in /signin and changed "classname" -> "className" in DataTable.tsx.
Fixed https://github.com/chester-hill-solutions/callcaster/issues/88
Updated /signup styles to be more consistent with /signin

### CODE REVIEW

**Comment by manstephen** [comment]
I refactored the layout of Workspace Settings a few times, and the page content itself is limited to members with Admin or Owner permissions. An Admin or Owner can add users directly to the workspace if they know the user's username (spelling perfect). Obviously this needs to be changed later to be a search function by username, first name, etc... 

Clicking on a gear next to each member will allow the owner/admin to open the member's details and:  
1. Change the member's workspace role (their permissions)
2. Remove the member from the workspace or
3. Transfer Ownership to the Member if the Owner is the active user

Admins cannot access the User Details of other Admins. Additionally right now the members are not sorted in the member list, and in the future I think it would be nice to sort them by role and then alphabetically. 

Bugs/Weaknesses:
1. More typing needs to be added into the system to better handle errors.
2. If the active user's permission becomes undefined and they try to access the page it'll crash (this is an ongoing issue with JWT refresh that will we fix.
3. Dark theme is automatically loaded when it matches system settings, but the conditional rendering of icons from react-icons doesn't work until dark theme is reselected using the theme selector.

TODOS:
1. Restricting Callers to specific campaigns (probably use a campaign_caller table)
2. Implement search feature for adding users to workspace more conveniently.
3. Implement filter feature to filter users already in the workspace? Might be nice if your workspace is like 100 people.

File: supabase/functions/invite-user-by-email/index.ts
Comment (wra-sol) [inline supabase/functions/invite-user-by-email/index.ts:1]: Test values in here.

File: .vscode/settings.json
Comment (manstephen) [inline .vscode/settings.json:1]: This is just for testing the edge function on my setup and does not need to go through

**Review by manstephen — COMMENTED** [review]
Everything should go through without trouble, except: 

1. Workspaces/id
2. Signup
3. Signin
4. Navbar

the .vscode settings and empty /audiences file also don't need to go through

**Review by wra-sol — CHANGES_REQUESTED** [review]
Remove
Workspaces/id
Signup
Signin
Navbar

**Review by wra-sol — APPROVED** [review]

**Review by wra-sol — APPROVED** [review]

**Review by wra-sol — APPROVED** [review]

**Review by wra-sol — APPROVED** [review]

**Pull request merged**